validate/validate.py

```diff
@@ -86,27 +88,32 @@ def _check_extension(extension:str):
 
     return UNKNOWN_FILE_TYPE
 
+def _validation_worker(path: Path, args:Union[ValidateArgs,Dict[str, Union[str,list]]]):
+    ''' Worker function to validate a single file '''
+    try:
+        file_type, file_extension = _detect_file_type_and_extension(path)
+        _validate_file(path, file_type, file_extension, args)
+    except FileNotFoundError as file_not_found_err:
+        print(f"Warning: {str(path)} {str(file_not_found_err)}")
+    except (TypeError, ValueError, IOError, OSError) as err:
+        _print_error(path, err)
```

Comment:
It's not a _huge_ concern, but if you're printing tracebacks from multiple processes the outputs can get interleaved. You might want a lock to prevent that: https://docs.python.org/3/library/multiprocessing.html#synchronization-between-processes

Reply:
I'll create an issue for this as it'll require a little more effort to put all print statements within the lock without affecting the parallelization itself

Reply:
Totally fair!

Reply:
@yashpatel6 for now, can you prepend the PID or some other kind of identifier?

Reply:
For future consideration, using the log utilities can help us here as well.

Reply:
Prepended the PID and added the log utilities to the issue